components/UserInfo/UserInfo.tsx

```diff
@@ -1,18 +1,7 @@
 import styles from './UserInfo.module.css';
 
-export default function UserInfo({
```

Comment:
if you dont like the inline typing thats fine, but throw it into an interface instead of using any!